blocks/extra-large-manual-promo-block/themes/news.json

```diff
@@ -28,49 +28,22 @@
 						"justify-content": "center",
 						"line-height": "var(--heading-level-1-line-height)"
 					},
+					"link-hover": {
+						"color": "var(--text-color-subtle)",
+						"opacity": 0.8
+					},
 					"overline": {
 						"display": "flex",
 						"justify-content": "center"
 					},
+					"overline-hover": {
+						"color": "var(--text-color-subtle)"
+					},
 					"stack": {
 						"gap": "var(--global-spacing-5)"
 					}
 				}
 			}
 		}
-	},
-	"xl-manual-promo-overline-hover": {
-		"styles": {
-			"default": {
-				"components": {
-					"overline-hover": {
-						"color": "var(--text-color-subtle)"
-					}
-				}
-			},
-			"desktop": {}
-		}
-	},
-	"xl-manual-promo-headline-hover": {
-		"styles": {
-			"default": {
-				"components": {
-					"heading": {
-						"color": "var(--text-color-subtle)"
-					}
-				}
-			},
-			"desktop": {}
-		}
-	},
-	"xl-manual-promo-img": {
-		"styles": {
-			"default": {
-				"link-hover": {
-					"opacity": 0.8
-				}
-			},
-			"desktop": {}
-		}
```

Comment:
Sorry if my comments caused some confusion, but we still need a token for the image container. The `opacity` property affects text as well, so the links are too light.

The change needed here was to add the components wrapper like so:
```
"xl-manual-promo-img": {
    "styles": {
        "default": {
            "components": {
	        "link-hover": {
		    "opacity": 0.8
		}
            }
        },
        "desktop": {}
    }
}
```

Reply:
@vgalatro, Fixed it in [a622ba2](https://github.com/WPMedia/arc-themes-blocks/pull/1690/commits/a622ba26042fa29d7ded26bda4f557df321862c0)